honeybee_radiance_folder/folder.py

```diff
@@ -0,0 +1,584 @@
+# -*- coding: utf-8 -*-
+"""
+Radiance folder structure module.
+
+This module includes classes for writing and reading to / from a Radiance folder
+structure.
+
+See https://github.com/ladybug-tools/radiance-folder-structure#radiance-folder-structure
+
+"""
+import os
+import json
+import itertools
+from .folderutil import parse_dynamic_apertures, parse_dynamic_geometries
+
+
+class _FolderCollection(object):
+    """Folder collection base class."""
+    REQUIRED = ()
+    __slots__ = ('_folder',)
+
+    def __init__(self, folder):
+        """Radiance folder  base class structure.
+
+        Args:
+            folder: Project folder. The folder will be created on write if it doesn't
+            exist already.
+        """
+        self._folder = os.path.normpath(folder)
+
+    @property
+    def folder(self):
+        """Return project folder."""
+        return self._folder
+
+    # TODO: extend validation
+    def validate(self):
+        """Validate a model folder."""
```

Comment:
You might have misunderstood what I was commenting on.  You say in that docstring that it validates a `model` folder when, in fact, it validates many different types of folders since it is a part of the base class.

Reply:
My bad! you are right. I will fix that.